### PR TITLE
PMDK release 2.1.0 w/o NDCTL enabled by default

### PR DESCRIPTION
**Pull request by osalyk**

### CODE REVIEW

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
Your PR title is saying the version to build is 2.1.0 but https://github.com/daos-stack/pmdk/blob/27ef152155d82b128d198690d9d9cdfa22862dc5/pmdk.spec#L18-L20 will result in 2.0.0 being built.

Additionally, every `*.spec` file change needs a `%changelog` entry and either a `Version:` update and/or a `Release:` update as well.

File: pmdk.spec
Comment (grom72) [inline pmdk.spec:1]: `# --with ndctl` instead of `# --with | --without ndctl`

File: pmdk.spec [hunk 1]
```diff
@@ -28,10 +28,10 @@
 
 %define min_ndctl_ver 60.1
 %define _make_common_args EXTRA_CFLAGS="-Wno-error" NORPATH=1 BUILD_EXAMPLES=n BUILD_BENCHMARKS=n
-%if %{without ndctl}
-    %define make_common_args %{_make_common_args} NDCTL_ENABLE=n
-%else
+%if %{with ndctl}
     %define make_common_args %{_make_common_args}
+%else
+    %define make_common_args %{_make_common_args} NDCTL_ENABLE=n
```
Comment (grom72) [inline pmdk.spec]:
```
%define make_common_args %{_make_common_args} NDCTL_ENABLE=n PMEMOBJ_IGNORE_DIRTY_SHUTDOWN=y PMEMOBJ_IGNORE_BAD_BLOCKS=y
```

**Review by grom72 — CHANGES_REQUESTED** [review]

**Comment by grom72** [comment]
> Your PR title is saying the version to build is 2.1.0 but
> 
> https://github.com/daos-stack/pmdk/blob/27ef152155d82b128d198690d9d9cdfa22862dc5/pmdk.spec#L18-L20
> 
> will result in 2.0.0 being built.
> Additionally, every `*.spec` file change needs a `%changelog` entry and either a `Version:` update and/or a `Release:` update as well.

Fixed by rebasing on the top of #37

File: pmdk.spec (hunk 1 above)
Comment (osalyk) [reply pmdk.spec]: Done.

File: pmdk.spec
Comment (osalyk) [reply pmdk.spec:1]: Done.

File: pmdk.spec [hunk 2]
```diff
@@ -28,10 +28,11 @@
 
 %define min_ndctl_ver 63
```
Comment (grom72) [inline pmdk.spec:29]:
```
# by default build without ndctl, unless explicitly enabled
%bcond_with ndctl
```

**Review by grom72 — CHANGES_REQUESTED** [review]

File: pmdk.spec (hunk 2 above)
Comment (osalyk) [reply pmdk.spec:29]: Done.

**Pull request merged**